app/api/entities/scan_task_entity.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+module Entities
+  class ScanTaskEntity < ApplicationEntity
+    expose(
+      :additional_note,
+      :additional_note,
+      :created_at,
+      :created_at,
+      :description,
+      :description,
+      :id,
+      :image,
+      :measurement_unit,
+      :measurement_value,
+      :private_note,
+      :sample_id,
+      :sample_svg_file,
+      :short_label,
+      :showed_name,
+      :status,
+      :updated_at
+    )
+    expose_timestamps
+
+    def sample_id
```

Comment:
The methods sample_id, sample_svg_file, showed_name and short_label can be replaced by using a delegate to sample